qa/integration-tests/src/test/java/io/camunda/zeebe/it/smoke/ContainerClusterSmokeIT.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.smoke;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.qa.util.testcontainers.ZeebeTestContainerDefaults;
+import io.zeebe.containers.cluster.ZeebeCluster;
+import io.zeebe.containers.engine.ContainerEngine;
+import java.util.concurrent.TimeUnit;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Testcontainers
+final class ContainerClusterSmokeIT {
+
+  private final ZeebeCluster cluster =
+      ZeebeCluster.builder()
+          .withBrokersCount(1)
+          .withGatewaysCount(1)
+          .withPartitionsCount(1)
+          .withEmbeddedGateway(false)
+          .withImage(ZeebeTestContainerDefaults.defaultTestImage())
+          .build();
+
+  @Container
+  private final ContainerEngine engine = ContainerEngine.builder().withCluster(cluster).build();
```

Comment:
That said, if we expand the test to also deploy/complete, then it might make sense to keep the `ContainerEngine` and use the assertions provided by `zeebe-process-test` :+1:

Reply:
Thanks for the input! I would go for the simpler approach for now.